src/repr/src/adt/mz_acl_item.rs

```diff
@@ -377,6 +379,123 @@ impl Columnation for MzAclItem {
     type InnerRegion = CopyRegion<MzAclItem>;
 }
 
+/// An encoded packed variant of [`MzAclItem`].
+///
+/// We uphold the variant that [`PackedMzAclItem`] sorts the same as [`MzAclItem`].
+#[derive(Debug, Copy, Clone, PartialEq, Eq, PartialOrd, Ord)]
+#[repr(align(8))]
+pub struct PackedMzAclItem([u8; Self::SIZE]);
+
+impl PackedMzAclItem {
+    // Note: It's critical to the sort order guarantee that these tags sort the same as the
+    // variants of RoleId.
+    //
+    // Note: While a u32 wastes a bit of memory it provides better memory alignment.
+
+    pub const SYSTEM_TAG: u32 = 100;
```

Comment:
Where'd these numbers come from, leaving room in between them for new values? Would be good to leave a comment here